### PR TITLE
chore: add css documentation for combobox, container, tooltip, timeline, variables and wizard

### PR DESCRIPTION
**Pull request by manjari23**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: projects/angular/src/utils/variables/API.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+| --clr-global-on-selection-color | When an element is chosen, it changes the colour of the text inside the element and is used globally.
+| --clr-global-hover-color | On mouse over, it changes the background colour of an element.
+| --clr-global-content-header-font-color | It changes the colour of the text for all the headings and is used globally.
+| --clr-global-font-color | It modifies the text colour of the element's description and is used globally.
+| --clr-global-success-color | On success, it sets the foreground colour value of an element's text.
+| --clr-global-error-color | On error, it sets the foreground colour value of an element's text.
+| --clr-close-color--normal | It sets the colour of the close icon.
+| --clr-close-color--normal-opacity | sets the level of transparency for an element.
+| --clr-close-color--hover | It sets the colour of the close icon when the mouse cursor is over it
+| --clr-close-color--hover-opacity | sets how transparent an element is when the mouse is over it.
```
Comment (Jinnie) [inline projects/angular/src/utils/variables/API.md:19]: It sets the transparency of the close icon on hover

File: projects/angular/src/utils/variables/API.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| --clr-popover-box-shadow-color | It changes one or more shadows on an element and used globally.
```
Comment (Jinnie) [inline projects/angular/src/utils/variables/API.md:20]: It sets the color of the highlight of the close icon

File: projects/angular/src/wizard/API.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Wizard-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-wizard-main-bgcolor | Changes the background color in wizard element.
+| --clr-wizard-stepnav-bgcolor | Changes the background color of the side navigation bar for all wizard elements..
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:6]: ..  -> .

Everywhere in the reviewed files, please

File: projects/angular/src/wizard/API.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-main-text-color | Changes the color of text in form validation for successfully filling information; in alt net, changes the color of text for checkbox text and in step navigation changes the text color of required information to proceed.
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:7]: Just "Wizard main text color" would be fine, thanks

File: projects/angular/src/wizard/API.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-stepnav-text | Changes the color of text inside the side navigation bar of wizard element for visited items.
+| --clr-wizard-stepnav-text--active | Changes the text's color for the currently active element inside the side nav bar of wizard element.
+| --clr-wizard-title-text | Changes the color of the text in the title of a wizard element. This change affects all wizard elements.
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:10]: Drop "The change affects all wizard elements"

File: projects/angular/src/wizard/API.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-stepnav-border-size | Changes the side nav bar's border size and is applied to each side nav bar of the wizard element.
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:11]: drop "and is applied to...." for this and all following items

File: projects/angular/src/wizard/API.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-stepnav-item-border-color | Changes the border color of the side nav bar and is applied to all wizard element side nav bars.
+| --clr-wizard-stepnav-item-border-color--active | Changes the border color of the side nav bar when element is active and is applied to all wizard element side nav bars.
+| --clr-wizard-stepnav-active-bgcolor | Changes the background color of an active item in a side navigation bar and applied it to all side navigation bars of wizard element.
+| --clr-wizard-header-action-color| It changes the color of the text.
+| --clr-wizard-header-action-color--hovered | when the mouse is over it, the text's color changes.
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:16]: Changes the text color on hover

File: projects/angular/src/wizard/API.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-border-radius | Changes the wizard element's corner radius and applies to all wizard elements..
+| --clr-wizard-stepnav-active-border-radius | Changes the side navigationcorner radius and applies to all side navigation bar.
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:18]: navigationcolor -> navigation color

File: projects/angular/src/wizard/API.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-stepnav-border-color | Changes the color of the side navigation bar's border in the wizard element..
+| --clr-wizard-box-shadow-color | Changes the color of the shadow surrounding an element's frame for all wizard elements..
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:20]: Changes the highlight color

File: projects/angular/src/wizard/API.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+
+## Class names
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-wizard | Has a predefined set of pages and navigation between them. |
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:26]: Main Wizard element

File: projects/angular/src/wizard/API.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+| clr-wizard-demo |  This is a container class for wizard.The purpose of container class is to contain, pad, and (occasionally) centre their contents. |
```
Comment (Jinnie) [inline projects/angular/src/wizard/API.md:27]: remove pls, this is demo app only class

File: projects/angular/src/wizard/API.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+| clr-wizard-basic | Class that encapsulates wizard. |
+| clr-wizard-content | Class wraps up the discription in wizard.|
+| clr-wizard-title | Class contains the wizard title. |
+| clr-wizard-stepnav-wrapper | This is a wrapper class which wraps wizard side navbar. |
+| clr-wizard-stepnav-list | Class contains the series of menu items. |
+| clr-wizard-stepnav-item | The class contains a list of menu items. |
+| clr-wizard-footer | Contains the wizard's footer section. |
+| clr-wizard-header-actions | Class let the client and the server pass additional information with an HTTP request or response. |
```
[the remark this answers is not in the document]
Comment (kevinbuhmann) [reply projects/angular/src/wizard/API.md:35]: A CSS class has nothing to do with HTTP requests...

File: projects/angular/src/wizard/API.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+## Class names
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-wizard | Has a predefined set of pages and navigation between them. |
+| clr-wizard-demo |  This is a container class for wizard.The purpose of container class is to contain, pad, and (occasionally) centre their contents. |
+| clr-wizard-basic | Class that encapsulates wizard. |
+| clr-wizard-content | Class wraps up the discription in wizard.|
+| clr-wizard-title | Class contains the wizard title. |
+| clr-wizard-stepnav-wrapper | This is a wrapper class which wraps wizard side navbar. |
+| clr-wizard-stepnav-list | Class contains the series of menu items. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/wizard/API.md:32]: Hi Jinnie, could you please mention the line number .

File: projects/angular/src/layout/main-container/API.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+| --clr-header-title-font-family | changes the font type for an element and is applied to all fonts in the header element.
+
+## Class names
+
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-header | The header class includes header elements. A header element usually contains one or more heading elements, such as a logo or an icon. |
+| header-hamburger-trigger | Class is used to toggle the visibility of the navigation bar |
+|  header-actions | In the header section, the class contains the dropdown, and the dropdown contains link items. |
+| clr-nav-demo | This is a container class.The purpose of container class is to contain, pad,   and (occasionally) centre their contents. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/layout/main-container/API.md:28]: Sure ,I will remove

File: projects/angular/src/utils/variables/API.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-global-borderradius | It changes the radius of the elemnt's corner and is applied globally.
+| --clr-global-borderwidth | It changes the width of the border and is used globally.
+| --clr-global-app-background | It changes the background colour of the body and is used globally.
+| --clr-global-selection-color | It changes the background colour when the element is selected and is used globally.
+| --clr-global-on-selection-color | It changes the background colour when the element is selected and is used globally.
+| --clr-global-on-selection-color | When an element is chosen, it changes the colour of the text inside the element and is used globally.
+| --clr-global-hover-color | On mouse over, it changes the background colour of an element.
+| --clr-global-content-header-font-color | It changes the colour of the text for all the headings and is used globally.
+| --clr-global-font-color | It modifies the text colour of the element's description and is used globally.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md:13]: should I change it to   "Changes the text color" ?

[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md:13]: Shall I change it to "global font color" ?

File: projects/angular/src/wizard/API.md (hunk 11 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md:27]: Sure,I will remove all demo classes.

File: projects/angular/src/forms/combobox/API.md [hunk 16]
```diff
@@ -0,0 +1,27 @@
+# Combobox-clarity - Style API
+
+| CSS Variable Name            | Description                                                 |
+| ---------------------------- | -------------------------------------------------------------|
+| --clr-combobox-trigger-color | Changes the color of an element.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: Updated ,as suggested

File: projects/angular/src/forms/combobox/API.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+| --clr-combobox-font-size     | Changes the combobox size .
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: Updated, as suggested.

File: projects/angular/src/forms/combobox/API.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+| --clr-combobox-border-color  | Changes the border color of the combobox.
+| --clr-combobox-border-radius | Changes the radius of the corners of the combobox.
+| --clr-combobox-input-background-color | Changes the color of the background.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: Updated, as suggested.
Thank you

File: projects/angular/src/forms/combobox/API.md [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+| --clr-combobox-pill-background-color  | Changes the background color of the selected option in a multiselect combobox.
+| --clr-combobox-pill-border-color | Changes the border color of the selected option in a multiselect combobox .
+| --clr-combobox-pill-border-radius | Changes the radius of the corners of the selected option in a multiselect combobox and is applied to favourite states and favourite states(Async).
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: updated, as suggested

File: projects/angular/src/forms/combobox/API.md [hunk 20]
```diff
@@ -0,0 +1,27 @@
...
+| --clr-combobox-pill-font-color | Changes text color of selected option in a multiselect combobox and is applied to favourite states and favourite states(Async).
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: updated, as suggested

File: projects/angular/src/wizard/API.md [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-box-shadow-color | Changes the color of the shadow surrounding an element's frame for all wizard elements..
+
+
+## Class names
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-wizard | Has a predefined set of pages and navigation between them. |
+| clr-wizard-demo |  This is a container class for wizard.The purpose of container class is to contain, pad, and (occasionally) centre their contents. |
+| clr-wizard-basic | Class that encapsulates wizard. |
+| clr-wizard-content | Class wraps up the discription in wizard.|
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated, as suggested

File: projects/angular/src/wizard/API.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated, as suggested.

File: projects/angular/src/wizard/API.md [hunk 22]
```diff
@@ -0,0 +1,36 @@
...
+| clr-wizard-title | Class contains the wizard title. |
+| clr-wizard-stepnav-wrapper | This is a wrapper class which wraps wizard side navbar. |
+| clr-wizard-stepnav-list | Class contains the series of menu items. |
+| clr-wizard-stepnav-item | The class contains a list of menu items. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated, as suggested

File: projects/angular/src/wizard/API.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Hi Jennie , I updated as suggested by you.
Hi Kevin, I have updated the description.
Thank you

File: projects/angular/src/forms/combobox/API.md [hunk 23]
```diff
@@ -0,0 +1,27 @@
...
+| --clr-combobox-filter-highlight | Changes  blur or color shifting on an element's rendering before the element is displayed.
+
+ 
+## Class names
+
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-combobox-container | This is a container class |
+| clr-combobox-form-control | This class enables elements to be upgraded with custom styles, sizing, focus states, and other features |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: Updated, As suggested

File: projects/angular/src/forms/combobox/API.md [hunk 24]
```diff
@@ -0,0 +1,27 @@
...
+| clr-combobox | The class consists of two elements, a drop-down menu and a text input |
+| clr-combobox-wrapper | class is used to put the content inside |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: Updated, As suggested

File: projects/angular/src/forms/combobox/API.md [hunk 25]
```diff
@@ -0,0 +1,27 @@
...
+| clr-combobox-input-wrapper | Class Input Wrapper is used to wrap input elements |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: Updated, As suggested

File: projects/angular/src/forms/combobox/API.md [hunk 26]
```diff
@@ -0,0 +1,27 @@
...
+| clr-combobox-input | Class includes the input element |
+| clr-combobox-trigger | Class includes a button with an icon for opening and closing |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/forms/combobox/API.md]: Updated, as suggested

File: projects/angular/src/layout/main-container/API.md [hunk 27]
```diff
@@ -0,0 +1,30 @@
+# Header-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-header-bg-color | Changes the background color of the first header in the header's colors.
+| --clr-header-divider-opacity | Changes the transparency of the header element
+| --clr-header-nav-opacity | Changes the header's transparency and applies the change to the top header of the page.
+| --clr-header-nav-hover-opacity | changes the header's transparency when the mouse is over it.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/layout/main-container/API.md]: Updated, as suggested

File: projects/angular/src/layout/main-container/API.md [hunk 28]
```diff
@@ -0,0 +1,30 @@
+# Header-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-header-bg-color | Changes the background color of the first header in the header's colors.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/layout/main-container/API.md]: Updated, as suggested

File: projects/angular/src/layout/main-container/API.md [hunk 29]
```diff
@@ -0,0 +1,30 @@
...
+| --clr-header-font-color | Changes the color of the text inside the header element and is applied to all headers in the header's color.
+| --clr-header-title-color | Changes the color of the header's title text and is applied to all headers in the header's colors.
+| --clr-header-title-font-weight | Changes the thickness of the text in the header element and is applied to all text in the header class.
+| --clr-header-title-font-family | changes the font type for an element and is applied to all fonts in the header element.
+
+## Class names
+
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-header | The header class includes header elements. A header element usually contains one or more heading elements, such as a logo or an icon. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/layout/main-container/API.md]: Updated, as suggested.

File: projects/angular/src/layout/main-container/API.md [hunk 30]
```diff
@@ -0,0 +1,30 @@
...
+| header-hamburger-trigger | Class is used to toggle the visibility of the navigation bar |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/layout/main-container/API.md]: Updated, As suggested.

File: projects/angular/src/layout/main-container/API.md [hunk 31]
```diff
@@ -0,0 +1,30 @@
...
+|  header-actions | In the header section, the class contains the dropdown, and the dropdown contains link items. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/layout/main-container/API.md]: Updated, As suggested

File: projects/angular/src/layout/main-container/API.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/layout/main-container/API.md]: Demo class I have removed.
Thanks

File: projects/angular/src/popover/tooltip/API.md [hunk 32]
```diff
@@ -0,0 +1,18 @@
+# Tooltip-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-tooltip-background-color | Changes the tooltip's background color and is applied to all tooltip elements.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/popover/tooltip/API.md]: Updated, As suggested

File: projects/angular/src/popover/tooltip/API.md [hunk 33]
```diff
@@ -0,0 +1,18 @@
...
+| --clr-tooltip-border-radius | Changes the tooltip element's corner radius and is applied to all tooltip elements.
+| --clr-tooltip-color | Changes the color of the text inside a tooltip element. This change affects all tooltip elements.
+| --clr-tooltip-font-weight | changes the thickness of text characters and is applied to all text contained within tooltip elements
+
+
+## Class names
+
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-tooltips-demo | This is a container class for tooltip.The purpose of container class is to contain, pad, and (occasionally) centre their contents. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/popover/tooltip/API.md]: Removed, As suggested

File: projects/angular/src/popover/tooltip/API.md [hunk 34]
```diff
@@ -0,0 +1,18 @@
...
+| clr-tooltips-sizes-demo | Class contains the various sizes of tooltips. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/popover/tooltip/API.md]: Removed, As suggested

File: projects/angular/src/popover/tooltip/API.md [hunk 35]
```diff
@@ -0,0 +1,18 @@
...
+| tooltip | Class includes tooltip element. When the user moves the mouse pointer over an tooltip element, extra information about that element is specified. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/popover/tooltip/API.md]: Updated, As suggested

File: projects/angular/src/popover/tooltip/API.md [hunk 36]
```diff
@@ -0,0 +1,18 @@
...
+| tooltip-content | Class displays the text in the tooltip |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/popover/tooltip/API.md]: Updated , as suggested

File: projects/angular/src/timeline/API.md [hunk 37]
```diff
@@ -0,0 +1,25 @@
+# Timeline-clarity - Style API
+
+| CSS Variable Name                | Description                                                  |
+| -------------------------------- | -------------------------------------------------------|
+| --clr-timeline-line-color        | Changes the color of the line in a timeline element.
+| --clr-timeline-step-header-color | Changes the text color in the header of each timeline element.
+| --clr-timeline-step-title-color  | Changes the text color of title for all static style and angular components of the timeline.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/timeline/API.md]: Updated, As suggested.

File: projects/angular/src/timeline/API.md [hunk 38]
```diff
@@ -0,0 +1,25 @@
...
+| --clr-timeline-step-description-color | Changes the text color of discription for all static style and angular components of the timeline.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/timeline/API.md]: Updated, As suggested

File: projects/angular/src/timeline/API.md [hunk 39]
```diff
@@ -0,0 +1,25 @@
...
+| --clr-timeline-current-step-color     | Changes the color of the current step circle in the timeline.
+| --clr-timeline-success-step-color     | Changes the color of the completed step circle in the timeline.
+| --clr-timeline-error-step-color       | Changes the color of the error step circle in the timeline element.
+| --clr-timeline-step-title-font-weight | Changes the thickness of the text characters in the description of timeline elements.
+
+## Class names
+
+| Class name               | Description                              |
+| ------------------------ | ---------------------------------------- |
+| clr-timeline-demo        | This is a container class |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/timeline/API.md]: Removed, As suggested.

File: projects/angular/src/timeline/API.md [hunk 40]
```diff
@@ -0,0 +1,25 @@
...
+| clr-timeline-demo-static | Class that includes each timeline component |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/timeline/API.md]: Removed demo class ,as suggested

File: projects/angular/src/timeline/API.md [hunk 41]
```diff
@@ -0,0 +1,25 @@
...
+| clr-timeline             | An unordered list class |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/timeline/API.md]: Updated, As suggested

File: projects/angular/src/timeline/API.md [hunk 42]
```diff
@@ -0,0 +1,25 @@
...
+| clr-timeline-step        | Class that shows a series of content for timeline |
+| clr-timeline-step-header | Class that displays content above the timeline in horizontal mode and to the left of the timeline in vertical mode |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/timeline/API.md]: Updated, As suggested

File: projects/angular/src/timeline/API.md [hunk 43]
```diff
@@ -0,0 +1,25 @@
...
+| clr-timeline-step-body   | Class that displays content below the timeline in horizontal mode and to the right of the timeline in vertical mode |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/timeline/API.md]: Updated, As suggested

File: projects/angular/src/utils/variables/API.md [hunk 44]
```diff
@@ -0,0 +1,20 @@
+# Global properties-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-global-borderradius | It changes the radius of the elemnt's corner and is applied globally.
+| --clr-global-borderwidth | It changes the width of the border and is used globally.
+| --clr-global-app-background | It changes the background colour of the body and is used globally.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md]: Sure, also I changed  to color

File: projects/angular/src/utils/variables/API.md [hunk 45]
```diff
@@ -0,0 +1,20 @@
...
+| --clr-global-selection-color | It changes the background colour when the element is selected and is used globally.
+| --clr-global-on-selection-color | It changes the background colour when the element is selected and is used globally.
+| --clr-global-on-selection-color | When an element is chosen, it changes the colour of the text inside the element and is used globally.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md]: I removed the duplicate.
Thank you

File: projects/angular/src/utils/variables/API.md [hunk 46]
```diff
@@ -0,0 +1,20 @@
...
+| --clr-global-hover-color | On mouse over, it changes the background colour of an element.
+| --clr-global-content-header-font-color | It changes the colour of the text for all the headings and is used globally.
+| --clr-global-font-color | It modifies the text colour of the element's description and is used globally.
+| --clr-global-success-color | On success, it sets the foreground colour value of an element's text.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md]: Updated, As suggested

File: projects/angular/src/utils/variables/API.md [hunk 47]
```diff
@@ -0,0 +1,20 @@
...
+| --clr-global-error-color | On error, it sets the foreground colour value of an element's text.
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md]: Updated, as suggested

File: projects/angular/src/utils/variables/API.md (hunk 2 above)
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md]: Updated ,As suggested

File: projects/angular/src/wizard/API.md (hunk 3 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Removed, and checked for other as well.

File: projects/angular/src/wizard/API.md (hunk 5 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated ,As suggested

File: projects/angular/src/wizard/API.md (hunk 4 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated, As suggested

File: projects/angular/src/wizard/API.md (hunk 6 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Dropped ,As suggested

File: projects/angular/src/wizard/API.md (hunk 8 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: navigationcorner ---> navigation corner updated

File: projects/angular/src/wizard/API.md (hunk 9 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated, As suggested

File: projects/angular/src/wizard/API.md (hunk 10 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated, As suggested

File: projects/angular/src/wizard/API.md (hunk 11 above)
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Demo classes I have removed.
Thanks

File: projects/angular/src/wizard/API.md [hunk 48]
```diff
@@ -0,0 +1,36 @@
...
+| clr-wizard-footer | Contains the wizard's footer section. |
+| clr-wizard-header-actions | Class let the client and the server pass additional information with an HTTP request or response. |
+| clr-wizard-stepnav |side navigation bar with links and menu items. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Updated, As suggested

File: projects/angular/src/wizard/API.md [hunk 49]
```diff
@@ -0,0 +1,36 @@
...
+| --clr-wizard-stepnav-border-color | Changes the color of the side navigation bar's border in the wizard element..
+| --clr-wizard-box-shadow-color | Changes the color of the shadow surrounding an element's frame for all wizard elements..
+
+
+## Class names
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-wizard | Has a predefined set of pages and navigation between them. |
+| clr-wizard-demo |  This is a container class for wizard.The purpose of container class is to contain, pad, and (occasionally) centre their contents. |
+| clr-wizard-basic | Class that encapsulates wizard. |
```
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/wizard/API.md]: Removed, As suggested

File: projects/angular/src/utils/variables/API.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (manjari23) [reply projects/angular/src/utils/variables/API.md]: Hi @Jinnie  Shall I change it to "Global font color" .z
Thank you

**Pull request closed without merging**